utils/node_local_test.py

```diff
@@ -1101,12 +1121,20 @@ def run_daos_client_cmd_pil4dfs(self, cmd, check=True, container=None):
             print('Stdout from command')
             print(rc.stdout.decode('utf-8').strip())
 
+        # if cwd and os.listdir(tmp_dir.name):
+        #    print('Temporary directory is not empty')
+        #    print(os.listdir(tmp_dir.name))
+        #    assert False, 'Files left in tmp dir by pil4dfs'
```

Comment:
Remove?

Reply:
Oh yeah, forgot about this.  If `pil4dfs` is working properly then the directory should be empty after use as all I/O is redirected to the container.  What is actually happening for this test is dfs_mount is failing and then pil4dfs is falling back to POSIX I/O and running the program against the backing POSIX file but unless there's a dfuse mount there then it should be rejecting the open call.  I'll file a bug Monday.